docs/spec/spec.md

```diff
@@ -155,11 +155,11 @@ channel._
 
 #### Spec
 
-| Field                  | Type                    | Description                                                                       | Constraints        |
-| ---------------------- | ----------------------- | --------------------------------------------------------------------------------- | ------------------ |
-| channel\*              | ObjectRef               | The originating _Subscribable_ for the link.                                      | Must be a Channel. |
-| subscriber<sup>1</sup> | eventing.SubscriberSpec | Optional processing on the event. The result of subscriber will be sent to reply. |                    |
-| reply<sup>1</sup>      | ReplyStrategy           | The continuation for the link.                                                    |                    |
+| Field                  | Type        | Description                                                                       | Constraints        |
+| ---------------------- | ----------- | --------------------------------------------------------------------------------- | ------------------ |
+| channel\*              | ObjectRef   | The originating _Subscribable_ for the link.                                      | Must be a Channel. |
+| subscriber<sup>1</sup> | Destination | Optional processing on the event. The result of subscriber will be sent to reply. |                    |
+| reply<sup>1</sup>      | Destination | The continuation for the link.                                                    |                    |
```

Comment:
I don't want to make this change yet since we haven't really discussed this with the community. I think this is a bigger change, and could / should be tackled in the v1beta1 track. Thoughts?

Reply:
Pin in it for 0.10, and we can update post release after a chatty chat?

Reply:
So, since this is the Spec as things are right now, can we just keep this as ReplyStrategy?

Reply:
I think this is the spec as we want them to be, or we need to add a version to the spec.

Reply:
I added the pkg and duck in the Type to indicate they are not part of the this repo, thoughts?

Reply:
Yeah but this is the spec, not the docs for the impl. The spec could live anywhere